src/swapd/runtime.rs

```diff
@@ -551,15 +499,6 @@ impl Runtime {
             return Err(Error::Farcaster(msg));
         }
 
-        // check if message swap id matches internal swap id
-        if request.swap_id() != self.swap_id() {
-            return Err(Error::Farcaster(format!(
-                "Incorrect swap id: expected {}, found {}",
-                self.swap_id(),
-                request.swap_id(),
-            )));
-        }
```

Comment:
why can we be sure that this can't happen?

Reply:
I removed it, because it was a code duplicate.

Reply:
oops, of course 🙈